connectors/sources/mysql.py

```diff
@@ -317,6 +318,11 @@ def _mysql_client(self):
     def advanced_rules_validators(self):
         return [MySQLAdvancedRulesValidator(self)]
 
+    @classmethod
```

Comment:
why not a simple function in utils? or a regular method?

Reply:
I also think this is not necessary a connector-specific implementation: https://github.com/elastic/connectors-python/pull/721#discussion_r1158086080

Reply:
@wangch079 @tarekziade I've created a now a `hash_id` function in `utils.py`, which will also be used as default implementation inside `BaseDataSource` so you don't have to worry about it as someone building a new connector.

Reply:
I meant, you don't have to implement a `hash_id` method in `BaseDataSource` class